app/views/projects/issues/index.html.erb

```diff
@@ -5,11 +5,28 @@
   project: current_project
 } %>
 
+<% if @open_issue.present? %>
+  <%= render partial: 'issues/issue', locals: {
+    issue: @open_issue,
+    project: current_project
+  } %>
+<% else %>
+  <%= turbo_frame_tag('issue') {} %>
```

Comment:
This entire if/else coulbe be in the application layout (so it can be used in other places in the future. But if this brings any complication right now, it's ok to leave here and on the `visualizations/show`